app/javascript/components/login_form.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react";
+import axios from "axios";
+interface LoginFormProps {
+  path: string;
+  token: string;
+  redirectPath: string;
+}
+const LoginForm: React.FC<LoginFormProps> = ({ path, token, redirectPath }) => {
+  const [email, setEmail] = React.useState("");
+  const [password, setPassword] = React.useState("");
+  const [errors, setErrors] = React.useState("");
+
+  const SendForm = (e: any) => {
+    e.preventDefault();
+    axios({
+      method: "post",
+      url: path,
+      headers: {
+        "X-CSRF-Token": token
+      },
+      data: {
+        email: email,
+        password: password
+      }
+    })
+      .then((response) => {
+        if (response.status === 200 && response.data.errors === undefined) {
+          window.location.href = redirectPath;
+        }
+        setErrors(response.data.errors);
+      })
+      .catch(() => {
+        window.alert("Nope");
+      });
+  };
+
+  return (
+    <div>
+      <form className="login-form" onSubmit={SendForm}>
+        <h1>LOGIN</h1> <br></br>
```

Comment:
More linebreak tags. It's not a blocker, just that we've discussed them before.

Reply:
Removed and added to the css file